tools/github-actions/nanomap-renderer-invoker.sh

```diff
@@ -9,7 +9,7 @@ cd "data/nanomaps"
 mv "cyberiad_nanomap_z1.png" "Cyberiad_nanomap_z1.png"
 mv "delta_nanomap_z1.png" "Delta_nanomap_z1.png"
 mv "MetaStation_nanomap_z1.png" "MetaStation_nanomap_z1.png"
```

Comment:
Can you tell all these lines exist just for consistency yet?

Reply:
a bit silly our maps are all inconsistent but what you gonna do, rename the map files?

Reply:
> rename the map files?

Please no Git will hate you